common_test.go

```diff
@@ -11,35 +12,39 @@ import (
 	"github.com/go-git/go-git/v5/plumbing/object"
 	"github.com/go-git/go-git/v5/storage/filesystem"
 	"github.com/go-git/go-git/v5/storage/memory"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
 
 	"github.com/go-git/go-billy/v5"
 	"github.com/go-git/go-billy/v5/memfs"
 	"github.com/go-git/go-billy/v5/osfs"
 	"github.com/go-git/go-billy/v5/util"
 	fixtures "github.com/go-git/go-git-fixtures/v4"
-	. "gopkg.in/check.v1"
 )
 
-func Test(t *testing.T) { TestingT(t) }
+type BaseFixtureSuite struct {
+	fixtures.Suite
+}
 
 type BaseSuite struct {
-	fixtures.Suite
+	suite.Suite
+	BaseFixtureSuite
 	Repository *Repository
 
 	cache map[string]*Repository
 }
 
-func (s *BaseSuite) SetUpSuite(c *C) {
-	s.buildBasicRepository(c)
+func (s *BaseSuite) SetupSuite() {
+	s.buildBasicRepository()
 
 	s.cache = make(map[string]*Repository)
 }
 
-func (s *BaseSuite) TearDownSuite(c *C) {
-	s.Suite.TearDownSuite(c)
-}
+// func (s *BaseSuite) TearDownSuite() {
+// 	s.Suite.TearDownSuite(c)
+// }
```

Comment:
We can remove this - I am working on a change that will remove the need for this call anyway.

```suggestion
```